comp/core/tagger/taggerimpl/collectors/workloadmeta_extract.go

```diff
@@ -435,6 +437,28 @@ func (c *WorkloadMetaCollector) handleKubeNode(ev workloadmeta.Event) []*types.T
 	return tagInfos
 }
 
+func (c *WorkloadMetaCollector) handleKubeNamespace(ev workloadmeta.Event) []*types.TagInfo {
+	namespace := ev.Entity.(*workloadmeta.KubernetesNamespace)
```

Comment:
Question:

`ev` is only used once to cast the entity to a namespace object.

after that, the `namespace` object is used only once below to build the tagger entity ID (line 451). 

I don't see where the entity data (i.e. the namespace labels and annotations) are being used in the tagger. To me it seems like the workloadmeta extractor in the tagger is always setting empty tag lists for all cardinalities for namespaces.

Am I missing something?

Reply:
Removed the changes in this file for now, to be added in again later (with actual functionality)

Reply:
If you remove `case workloadmeta.KindKubernetesNamespace:` the `default` will apply which logs an error.
I think it's better to do what we're doing with deployments and processes:
```go
case workloadmeta.KindKubernetesDeployment:
// tagInfos = append(tagInfos, c.handleDeployment(ev)...) No tags for now
```
That way the agent will not log unnecessary errors.